Directory.Build.props

```diff
@@ -7,4 +7,12 @@
   </PropertyGroup>
 
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+  <!-- Workaround to enable .editorconfig based analyzer configuration until dotnet compilers support .editorconfig based configuration -->
```

Comment:
How does this workaround work?

Reply:
@genlu Details here: https://github.com/dotnet/roslyn-analyzers/pull/1952#issue-241402641